README.md

```diff
@@ -32,6 +32,7 @@ The following Windows versions are known to work (built with VMware Fusion Pro
 * Windows Server 2022 Desktop -> Vagrant Cloud box [StefanScherer/windows_2022](https://app.vagrantup.com/StefanScherer/boxes/windows_2022)
 * Windows Server 2019 Desktop -> Vagrant Cloud box [StefanScherer/windows_2019](https://app.vagrantup.com/StefanScherer/boxes/windows_2019)
 * Windows Server Core
+  * Windows Server 2022 without and with Docker -> Vagrant Cloud box [StefanScherer/windows_2022_docker](https://app.vagrantup.com/StefanScherer/boxes/windows_2022_docker)
```

Comment:
I suggest to skip the link to a Vagrant Cloud box. My pipeline to build and upload such boxes is broken and I hadn't the time to investigate and fix it.

```suggestion
  * Windows Server 2022 without and with Docker
```